### PR TITLE
Remove accidental mismatch of "PAMAP"

### PR DESCRIPTION
**Pull request by cicku**
image

### CODE REVIEW

**Comment by mmd-osm** [comment]
https://www.openstreetmap.org/way/166528135 would no longer be flagged by this change, although it might be still incompatible data (edit: to be verified!).

Adding more tests in https://github.com/openstreetmap/iD/blob/develop/test/spec/validations/incompatible_source.js might be good.

**Comment by cicku** [comment]
I don’t know anything about Europe so I can’t help here. As far as I can see it’s a regression introduced by a patch to warn about proprietary Chinese data.

File: modules/validations/incompatible_source.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export function validationIncompatibleSource() {
   const incompatibleRules = [
     {
       id: 'amap',
-      regex: /(amap|autonavi|mapabc|高德)/i
+      regex: /(^amap$|amap\.at|^amap\.com|autonavi|mapabc|高德)/i
```
Comment (tyrasd) [inline modules/validations/incompatible_source.js]: Can you please drop `amap.at` from this list? It doesn't have anything to do with the data from [AutoNavi](https://en.wikipedia.org/wiki/AutoNavi), which this rule is [about](https://github.com/openstreetmap/iD/pull/8701#issuecomment-921922242). The amap.at domain is owned by an Austrian government agency.

@mmd-osm if you'd like to see this URL added to the list of incompatible sources, please open a new PR (or issue) for it. :blush: At first glance, amap.at seems to be a former, now dysfunctional, URL of what is now available at http://www.austrianmap.at/amap/, which AFAICS does contain non-open data (e.g., `ÖK 50`).

**Review by tyrasd — CHANGES_REQUESTED** [review]

File: modules/validations/incompatible_source.js (hunk 1 above)
Comment (cicku) [reply modules/validations/incompatible_source.js]: Done

**Review by tyrasd — APPROVED** [review]

**Pull request merged**